tests/containers/bci_test.pm

```diff
@@ -29,21 +29,29 @@ sub run_tox_cmd {
     my $bci_timeout = get_var('BCI_TIMEOUT', 1200);
     my $bci_reruns = get_var('BCI_RERUNS', 3);
     my $bci_reruns_delay = get_var('BCI_RERUNS_DELAY', 10);
-    my $cmd = "tox -e $env -- -n auto";
+    my $tox_out = "tox_output.txt";
+    my $cmd = "tox -e $env -- -rxX -n auto";
     $cmd .= " -k \"$bci_marker\"" if $bci_marker;
     $cmd .= " --reruns $bci_reruns --reruns-delay $bci_reruns_delay";
+    $cmd .= "| tee $tox_out";
+    # Cut the tox log from the header onward and filter the text
+    my $cmd_xf = "awk '/short test summary info/{f=1}f' $tox_out | grep XFAIL";
```

Comment:
Please move this variable declaration down to where it is needed. It is weird to have the variable being declared somewhere and then being used somewhere else.